velox/functions/prestosql/Probability.h

```diff
@@ -118,6 +119,26 @@ struct CauchyCDFFunction {
   }
 };
 
+template <typename T>
+struct LaplaceCDFFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+  FOLLY_ALWAYS_INLINE void
+  call(double& result, double location, double scale, double x) {
+    result = std::numeric_limits<double>::quiet_NaN();
+    if (std::isnan(location) || std::isnan(scale) || std::isnan(x)) {
```

Comment:
I think the previous result value should be set within the conditional. Its very atypical to have an empty conditional statement.